R/workhorse.R

```diff
@@ -75,22 +75,24 @@ make_request <- function(x, ...){
 }
 
 process_request <- function(res,
-                            content = TRUE,
-                            expected = "application/json; charset=UTF-8") {
-  httr::stop_for_status(res)
-  if (content == TRUE) {
-    actual <- res$headers$`content-type`
-    if (actual != expected) {
-      spf(
-        paste0(
-          "Expected content-type:\n%s",
-          "\n",
-          "Actual content-type:\n%s"
-        ),
-        expected,
-        actual
-      )
-    }
-    jsonlite::fromJSON(httr::content(res, "text"), simplifyVector = FALSE)
+                            expected = "application/json; charset=UTF-8",
+                            internet = TRUE) {
```

Comment:
What's the rationale for an `internet` argument here?

If it's testing related, which I'm guessing it is, let's see if we can find an approach that serves the same purpose that is contained to the tests.